lib/core-integration/src/Test/Integration/Scenario/API/ByronWallets.hs

```diff
@@ -71,12 +73,30 @@ import qualified Network.HTTP.Types.Status as HTTP
 spec :: forall t. (EncodeAddress t, DecodeAddress t) => SpecWith (Context t)
 spec = do
 
+    describe "BYRON_ESTIMATE_06 - non-existing wallets" $  do
+        forM_ falseWalletIds $ \(title, walId) -> it title $ \ctx -> do
+            let endpoint = "v2/byron/wallets/" <> T.pack walId <> "/migrate"
+            rg <- request @ApiByronWallet ctx ("GET", endpoint) Default Empty
+            expectResponseCode @IO HTTP.status404 rg
+            expectErrorMessage errMsg404NoEndpoint rg
+
     it "BYRON_GET_02 - Byron ep does not show new wallet" $ \ctx -> do
         w <- emptyWallet ctx
         let ep  = ( "GET", "v2/byron/wallets/" <> w ^. walletId )
         r <- request @ApiByronWallet ctx ep Default Empty
         expectResponseCode @IO HTTP.status501 r
 
+    describe "BYRON_GET_06 - non-existing wallets" $  do
+        forM_ falseWalletIds $ \(title, walId) -> it title $ \ctx -> do
+            let endpoint = "v2/byron/wallets/" <> T.pack walId
+            rg <- request @ApiByronWallet ctx ("GET", endpoint) Default Empty
+            if (title == "40 chars hex") then do
```

Comment:
Is the `"40 chars hex"` value a special value? If so, perhaps we can make it into a shared constant, and reference it from here? What do you think?

Reply:
Yes, the _underneath_ walletId value is a valid 40 char long hex encoded string, but for non-existing wallet. I have added a shared constant. (I think :sweat_smile: )